.github/workflows/gradle.yml

```diff
@@ -112,6 +124,21 @@ jobs:
         id: version_info
         run: echo version=$(./gradlew properties -q -Prelease.useLastTag=true | grep "version:" | awk '{print $2}') >> "$GITHUB_OUTPUT"
 
+      - name: Publish to NuGet
+        if: github.ref == 'refs/heads/main'
+        env:
+          NUGET_TOKEN: ${{ secrets.NUGET_TOKEN }}
+        run: |
+          ./gradlew mingwX64Binaries
```

Comment:
Does this need `-Prelease.useLastTag=true` as well?

Reply:
Yup, added!